src/librustc_middle/query/mod.rs

```diff
@@ -281,6 +281,8 @@ rustc_queries! {
             cache_on_disk_if { key.is_local() }
         }
 
+        /// The `DefId` is the `DefId` of the containing MIR body. Promoteds to not have their own
+        /// `DefId`.
```

Comment:
So this returns the MIR of *all* promoteds in one go? Maybe say that. The name sounds singular.